### PR TITLE
Update MeasurementLink documentation

### PR DESCRIPTION
**Pull request by pbirkhol-ni**
- [x] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/measurement-services-labview/blob/main/CONTRIBUTING.md). _(Required)_
- [ ]  Automatically post PR comments with images for G code changes? _(Recommended for small changes)_

### What does this Pull Request accomplish?

The MeasurementLink documentation is out of date. This PR updates the documentation to include many of the recent changes. It updates screenshots, where applicable (and deletes unused screenshots). Finally, it is pretty aggressive about deleting duplicate or extraneous documentation.

### CODE REVIEW

File: Source/Generator/README.md
Comment (pbirkhol-ni) [inline Source/Generator/README.md:1]: This README was basically a duplicate of the main README. No sense in keeping up to date identical information in two places.

File: docs/README.md [hunk 1]
```diff
@@ -33,101 +33,88 @@ The MeasurementLink Support for LabVIEW packages enable measurement developers t
...
-3. Add the measurement details in the block diagram of the `Get Measurement Details.vi` including:
-    - display name
-    - version
-    - Service Class
+3. Add the measurement details to `Get Measurement Details.vi`.
 
     ![Get Measurement Details VI](images/Get%20Measurement%20Details%20VI.png)
 
-4. Add the required controls for the configuration parameters in the `Measurement Configuration.ctl` file. By default, this comes with `Array in` which is a double array.
+4. Add the required configuration parameters to `Measurement Configuration.ctl`. By default, there is a single double array called `array in`.
```
Comment (dixonjoel) [inline docs/README.md:61]: 'single double' is awkward. Maybe just "there is a double array...."

File: docs/README.md [hunk 2]
```diff
@@ -33,101 +33,88 @@ The MeasurementLink Support for LabVIEW packages enable measurement developers t
...
 
     ![Measurement Configurations ctl](images/Measurement%20Configurations%20ctl.png)
 
-5. Add the required indicators for the output parameters in the `Measurement Results.ctl` file. By default, this comes with the `Array out` parameter which is a double array.
+5. Add the required output parameters to `Measurement Results.ctl`. By default, there is a single double array called `array out`.
```
Comment (dixonjoel) [inline docs/README.md:65]: 'single double'

**Review by dixonjoel — APPROVED** [review]
Approved with suggestions.

**Pull request merged**